designs/disruption-controls.md

```diff
@@ -0,0 +1,118 @@
+# Disruption Controls
+
+Karpenter is including new Disruption Controls in its `v1beta1` APIs. These APIs include a new `Disruption` block in the `NodePool` that will contain all scale-down behavioral fields. Initially, this will include `Budgets`, which define a (1) parallelism of how many nodes can be deprovisioned at a time and a (2) cron schedule that determines when the parallelism applies. Disruption Controls will also include a `ConsolidateAfter` field that will allow users to affect the speed at which Karpenter will scale down underutilized nodes.
+
+## Proposed Spec
+
+This only includes the `Budgets` and `ConsolidateAfter` portion of the disruption spec.
+
+```{yaml}
```

Comment:
Huh, I've never done brackets like this. Are these needed or is it just a style choice whether you have them or don't?

Reply:
I did it wrong here, but doing 
```
```go 
```
works at formats/highlights it for you!